aws/sentinel.hcl

```diff
@@ -136,5 +136,5 @@ policy "restrict-subnet-of-ec2-instances" {
 
 policy "validate-providers-from-desired-regions" {
   source = "./validate-providers-from-desired-regions.sentinel"
-  enforcement_level = "advisory"
+  enforcement_level = "soft-mandatory"
```

Comment:
@Rabattkarte is there a reason why you feel that we need to change the enforcement level for the policy?

Reply:
Actually no, apparently that commit slipped in my branch. I'm sorry for this. Let me amend this. My bad.